### ISSUE TITLE
roxygen tag for coverage

### ISSUE THREAD

**Issue by thebioengineer** [phuse-org/valtools#49]
`@coverage` <- similar to `@section`

```
#' @section Specification coverage:
#' 7.1: 007.7.1, 007.7.2 
#' 7.2: 007.7.1, 007.7.2 
```

**Comment by thebioengineer** [comment]
dynamic referencing:

```
#' @section Specification coverage:
#' ##tc:my_test_case.1: ##req:my_requirement.1, ##req:my_requirement.2 
#' ##tc:my_test_case.2: ##req:my_requirement.1, ##req:my_requirement.2 

```

**Comment by thebioengineer** [comment]
TODO: update dynamic referencing to ignore text in roxygen header comments when determining order of cases. in example above, ##tc:text_case_B would render as 2 right now, which is non-sensical.

**Comment by thebioengineer** [comment]
Coverage tag in #51 
Addressed dynamic referencing in #55
